velox/exec/tests/utils/SingleThreaded.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "SingleThreaded.h"
+
+namespace facebook::velox::exec::test {
+
+std::unique_ptr<folly::Executor> newSingleThreadedExecutor() {
```

Comment:
What is this for?

Reply:
I see. I find it very confusing that a function named newSingleThreadedExecutor returns nullptr.

I suggest not to include this change in this PR.

Feel free to open a new PR to update documentation.

CC: @bikramSingh91

Reply:
> I find it very confusing that a function named newSingleThreadedExecutor returns nullptr.

Ideally caller doesn't have to know how it's implemented but I'll remove if it feels like confusing anyway. I'll update the PR later. Thanks.